client/src/main/java/org/spine3/base/Identifiers.java

```diff
@@ -47,14 +47,45 @@ private Identifiers() {
      *
      * @param <I> the type of the ID
      * @param idClass the class of IDs
-     * @throws IllegalArgumentException if the class of IDs is not supported
+     * @throws IllegalArgumentException if the class of IDs is not of supported type
      */
-    public static <I> void checkSupported(Class<I> idClass) {
+    private static <I> void checkSupported(Class<I> idClass) {
```

Comment:
The private static one was left to be used during compile-time model check. But I'm going to move the check from `AbstractEntity` constructor into `Repository.getIdClass()`.

That would optimize the performance (as we won't have to check for id values all the time), and leave only one method.